### DISCUSSION TITLE
Docker sunsets the free Team Organization feature

### DISCUSSION THREAD

**Discussion by christyjacob4** [TimWolla/docker-adminer#139] [Q&A] [answered]
As you're aware, there has been a recent change in Docker's policy around their free Team Organization which will lead to all existing images to be deleted. More info here [https://news.ycombinator.com/item?id=35154025](https://news.ycombinator.com/item?id=35154025)

Since the `adminer` image is also [hosted on DockerHub](https://hub.docker.com/_/adminer), I wanted to check how this change will affect it and if we as consumers of the image should be concerned about it in any way or take any necessary steps.

**Comment by TimWolla** [comment, marked as answer]
> As you're aware

Based on that phrasing, I expected you to link to the HN submission where I already commented. But it appears there is a new one.

> I wanted to check how this change will affect it and if we as consumers of the image should be concerned about it in any way or take any necessary steps.

Please refer to my comments in this subthread (HN username = GitHub username): https://news.ycombinator.com/item?id=35168802

**Reply by christyjacob4** [reply]
Thanks a lot for the clarification @TimWolla 👍  That helps. 

From the FAQ docs, it looks like the official Docker images will not be affected. 

![Screenshot 2023-03-17 at 12 30 18 AM](https://user-images.githubusercontent.com/20852629/225726264-404a931e-5d7c-47ca-86e9-58c2628e4f2c.png)
